setup.py

```diff
@@ -70,6 +70,14 @@ def exit_with_error(message):
     INCLUDE_ZSTD = os.environ['INCLUDE_ZSTD'] == '1'
 except KeyError:
     INCLUDE_ZSTD = True
+try:
+    INCLUDE_SSE2 = os.environ['INCLUDE_SSE2'] == '1'
+except KeyError:
+    INCLUDE_SSE2 = True
+try:
+    INCLUDE_AVX2 = os.environ['INCLUDE_AVX2'] == '1'
+except KeyError:
+    INCLUDE_AVX2 = True
 
```

Comment:
There is kind of a standard-de-facto way to not add support for SSE2 or AVX2.  The idea is to set a `DISABLE_XXX_SSE2` and `DISABLE_XXX_AVX2` environment variables (XXX can be 'BLOSC' here).  See for example: https://github.com/Blosc/bcolz/blob/master/setup.py#L132.